src/features/deposit/components/DepositUIMachineProvider.tsx

```diff
@@ -105,34 +104,31 @@ export function DepositUIMachineProvider({
 
                 let tx: Transaction["NEAR"][] = []
 
+                const storageDepositPayment = await getNEP141StorageRequired({
+                  tokenAddress,
```

Comment:
Logic above (extracting address of the token) is not fully correct. It does not handle a case when token is native, and technically it doesn't have an address. You need to consider it as well.

Reply:
As far as I see it only deals with NEAR tokens, and when Native near is accepted in our code we work with it as wrap.near .

PLS correct me if I am wrong

Reply:
FungibleTokenInfo and NativeTokenInfo both have `address` property (for some specific reasons). In case of native token address is "native". 
> when Native near is accepted in our code we work with it as wrap.near .
Not correct, for example, during deposit we accept just NEAR